assembly/src/snapshots/miden_assembly__tests__assert_eqw_with_code.snap

```diff
@@ -37,5 +37,44 @@ begin
         assert(15491226248792286710)
         eq
         assert(15491226248792286710)
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
+        noop
```

Comment:
Without having looked into this in detail, something looks off here: there should never be these many padded noops in a basic block. As a very rough guess, I think no basic block should end up with more than 8 padded noops.